test/test_jit_cmaps.py

```diff
@@ -19,13 +19,15 @@ def test_cmap_triangle(compile_args):
     mesh = ufl.Mesh(element)
     compiled_cmap, module = ffcx.codegeneration.jit.compile_coordinate_maps(
         [mesh], cffi_extra_compile_args=compile_args)
-    x = np.array([[0.5, 0.5]], dtype=np.float64)
-    x_ptr = module.ffi.cast("double *", module.ffi.from_buffer(x))
-    X = np.zeros_like(x)
+
+    # Reference coordinates X
+    X = np.array([[0.5, 0.5]], dtype=np.float64)
     X_ptr = module.ffi.cast("double *", module.ffi.from_buffer(X))
+    x = np.zeros_like(X)
+    x_ptr = module.ffi.cast("double *", module.ffi.from_buffer(x))
     coords = np.array([0.0, 0.0, 2.0, 0.0, 0.0, 4.0], dtype=np.float64)
     coords_ptr = module.ffi.cast("double *", module.ffi.from_buffer(coords))
-    compiled_cmap[0].compute_reference_coordinates(X_ptr, X.shape[0], x_ptr, coords_ptr)
+    compiled_cmap[0].compute_reference_coordinates(x_ptr, x.shape[0], X_ptr, coords_ptr)
```

Comment:
Why like this? `ufc.h` says 
```c
    void (*compute_reference_coordinates)(
        double* restrict X, int num_points, const double* restrict x,
        const double* restrict coordinate_dofs);
```
where capitalized always means reference.

Reply:
That's what I thought too. I must have made a mistake.